yarn.lock

```diff
@@ -1,7059 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
No lock file?

Reply:
what kinda lock file?

Reply:
This one. `yarn.lock` is being deleted.

Reply:
ahh i dont wanna delete that. thanks :+1: